Tests/Event/Listenener/PrepareListenerTest.php

```diff
@@ -3,8 +3,9 @@
 namespace Lexik\Bundle\FormFilterBundle\Tests\Event\Listener;
 
 use Lexik\Bundle\FormFilterBundle\Event\Listener\PrepareListener;
+use PHPUnit\Framework\TestCase;
 
-class PrepareListenerTest extends \PHPUnit_Framework_TestCase
+class PrepareListenerTest extends TestCase
```

Comment:
If you want to switch to newer phpunit version, you must be sure to avoid deprecated functions. E.g., $this->getMock() has been removed